R/scm_support.R

```diff
@@ -44,38 +43,50 @@ equilibrium_quiet <- function(base=Control()) {
   base
 }
 
+##' Hopefully sensible set of parameters for use with the SCM.  Turns
+##' accuracy down a bunch, makes it noisy, sets up the
+##' hyperparameterisation that we most often use.
+##' @title Sensible, fast (ish) SCM parameters
+##' @author Rich FitzJohn
+##' @param type Name of model (defaults to FF16 but any strategy name is valid).
+##' @export
+scm_base_control <- function(type="FF16", env=environment_type(type)) {
```

Comment:
I wonder if we should remove the default type, to force that type needs to be specified?